app/assets/stylesheets/styles/pages/content-page.scss

```diff
@@ -21,6 +27,30 @@ $content-page-redlist-logo-height: rem-calc(80);
     @include h2-content-page;
     margin: 1.2em 0 0.4em;
   }
+
+  li, ol {
+    line-height: $body-line-height;
+
+    @include breakpoint_down($small) {
+      font-size: rem-calc(14);
+    }
+  }
+
+  &__video-container {
+    margin: rem-calc(20) 0;
```

Comment:
Can we increase the padding to give some more space between the container and the surrounding text?